apiserver/facades/agent/provisioner/provisioninginfo.go

```diff
@@ -264,14 +261,8 @@ func (api *ProvisionerAPI) machineVolumeParams(
 		if err != nil {
 			return nil, nil, errors.Errorf("getting volume %q parameters: %w", volumeTag.Id(), err)
 		}
-		if _, err := env.StorageProvider(storage.ProviderType(volumeParams.Provider)); errors.Is(err, jujuerrors.NotFound) {
-			// This storage type is not managed by the environ
-			// provider, so ignore it. It'll be managed by one
-			// of the storage provisioners.
-			continue
-		} else if err != nil {
-			return nil, nil, errors.Errorf("getting storage provider: %w", err)
-		}
+		// TODO(storage): Handle the case:
+		// environ.StorageProvider(storage.ProviderType(volumeParams.Provider))
```

Comment:
I prefer calls to stub service methods (on the target service), but this is fine for now.